src/main/ts/core/Utils.ts

```diff
@@ -0,0 +1,22 @@
+import { PredicateFilter, SugarElement } from "@ephox/sugar"
+
+const isNestedList = (selectedElement: SugarElement): boolean => {
+  return PredicateFilter.ancestors(
```

Comment:
When trying to determine a yes/no answer, _never_ calculate all of them, and then ensure the list is larger than 1. That won't short-circuit as soon as it finds one. The `PredicateExists` module exists for these sorts of binary decisions. 

That's sort of the naming convention in `sugar`

`*Exists`: -> returns true/false
`*Filter` -> returns arrays